src/skimage/_shared/utils.py

```diff
@@ -1096,3 +1097,121 @@ def as_binary_ndarray(array, *, variable_name):
                 f"safely cast to boolean array."
             )
     return np.asarray(array, dtype=bool)
+
+
+def _prescale_value_range(image, *, mode, stacklevel=3):
+    """Scale the value range of `image` according to the selected `mode`.
+
+    Parameters
+    ----------
+    image : ndarray
+        The image to scale.
+    mode : {'minmax', 'legacy', False} or tuple[float, float], optional
+        Controls the rescaling behavior for `image`.
+
+        ``'minmax'``
+            Normalize `image` between 0 and 1 regardless of dtype. After
+            normalization its minimum and maximum values will be 0 and 1
+            respectively. This is a shorthand for
+            ``prescale=(image.min(), image.max())``.
+
+        ``'legacy'``
+            Normalize only if `image` has an integer dtype, if `image` is of
+            floating dtype, it is left alone. See :ref:`.img_as_float` for
+            more details.
+
+        ``(lower, higher)``
```

Comment:
```suggestion
        ``(min, max)``
```

Reply:
I'm not sure I agree. The two passed values don't need to correspond to a minimum and maximum. They are just a two values – the first lower than the other – that will correspond to 0 and 1 after scaling.

I remember us introducing that option precisely for batch cases, where that might not be the case? Thinking about it, it's an assumption I'd like to avoid using in our documentation. 

Using `lower` and `higher` also avoids slight confusion with Python's `min()` and `max()` functions. But that's  minor thing.

Reply:
Ah, so you think we purposefully should allow scaling that takes the image outside of [0, 1]?